snap/validate_test.go

```diff
@@ -82,6 +82,23 @@ func (s *ValidateSuite) TestValidateEpoch(c *C) {
 	}
 }
 
+func (s *ValidateSuite) TestValidateLicense(c *C) {
+	validLicenses := []string{
+		"GPL-3.0", "(GPL-3.0)", "GPL-3.0+", "GPL-3.0 AND GPL-2.0", "GPL-3.0 OR GPL-2.0", "BSD OR (GPL-3.0 AND GPL-2.0)", "BSD OR(GPL-3.0 AND GPL-2.0)",
```

Comment:
`BSD` isn't valid in SPDX. Each BSD "flavor" is broken down into a separate license tag.